mysite/blog/views.py

```diff
@@ -20,3 +22,35 @@ def post_detail(request, year, month, day, post):
         publish__day=day,
     )
     return render(request, "blog/post/detail.html", {"post": post})
+
+
+def post_share(request, post_id):
```

Comment:
https://github.com/pravintp/my-blog/commit/54216cfdf7f4a5b3e2fa83622b5447c22cab635c#diff-ae5986e281ef46bd77930a4985d5c376ce41689b3f9b85dcae0e6b60c77eecc8R29

For more cleaner way use FormView